### PR TITLE
Conformance targets

### PR DESCRIPTION
**Pull request by tschmidtb51**
- addresses parts of #140
- remove conformance targets "Result management system", "Engineering system"
- add "CVRF CSAF converter" #154

### CODE REVIEW

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 1]
```diff
@@ -1856,22 +1859,133 @@ A program satisfies the "CVRF CSAF converter" conformance profile if:
...
+
+* It satisfies the "CSAF producer" conformance profile.
+* It satisfies the "CSAF viewer" conformance profile.
+* It provides at least the following management functions:
+
+  * create new CSAF documents
+  * prefill CSAF documents based on values given in the configuration (see below)
+  * create a new version of an existing CSAF document
+  * checkout old versions of a CSAF document
+  * show diff between versions of a CSAF document
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:1874]: Maybe `show differences`?

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 2]
```diff
@@ -1856,22 +1859,133 @@ A program satisfies the "CVRF CSAF converter" conformance profile if:
...
+  * list all CSAF documents within the system
+  * delete CSAF documents from the system
+  * review CSAF documents in the system
+  * approve CSAF documents
+  * search for CSAF documents by values of required fields at `document`-level or their children within the system
+  * search for CSAF documents by values of `cve` within the system
+  * search for CSAF documents based on properties of `product_tree`
+  * filter on all properties which it is required to search for
+  * export of CSAF documents
+  * show an autdit log for each CSAF document
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:1884]: Nit: `audit`

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 3]
```diff
@@ -1856,22 +1859,133 @@ A program satisfies the "CVRF CSAF converter" conformance profile if:
...
+
+* It must identify the latest version of CSAF documents with the same `/document/tracking/id`
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:1886]: Simply `identify` Instead of It must identify?

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 4]
```diff
@@ -1856,22 +1859,133 @@ A program satisfies the "CVRF CSAF converter" conformance profile if:
...
+* It suggests a `/document/tracking/id` based on the given configuration.
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:1887]: Similarly: `suggest` ...

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 5]
```diff
@@ -1856,22 +1859,133 @@ A program satisfies the "CVRF CSAF converter" conformance profile if:
...
+* It keeps track of the version of CSAF documents automatically and increments for each change at least the patch_version.
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:1888]: Suggest to replace `It keeps track` with `track`

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 6]
```diff
@@ -1856,22 +1859,133 @@ A program satisfies the "CVRF CSAF converter" conformance profile if:
...
+* It must support the following workflows:
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:1889]: Suggest to replace `It must support` with `support`

**Review by sthagen — APPROVED** [review]
These changes help a lot and despite some comments I made they look good to me and I can easily go over them in my pass after the PR is merged 👍🏽

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 1 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: :+1 I change that.

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 2 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: Thanks for catching it. I change it.

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 3 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: :+1 I change that.

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 5 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: :+1 I change that.

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 6 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: :+1 I change that.

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 4 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: :+1 I change that.

**Review by sthagen — APPROVED** [review]
Very helpful. LGTM. Will merge and then perform the overall edit pass

**Comment by tschmidtb51** [comment]
Also fixes: #164, #156, #183, #168, #172

**Pull request merged**